go.mod

```diff
@@ -526,7 +526,7 @@ require (
 	github.com/twmb/franz-go/pkg/kadm v1.9.0
 	github.com/twmb/franz-go/pkg/kmsg v1.6.1 // indirect
 	github.com/ugorji/go/codec v1.2.11 // indirect
-	github.com/ulikunitz/xz v0.5.11 // indirect
+	github.com/ulikunitz/xz v0.5.11
```

Comment:
Following the thread in #fleet-automation-updates, can we continue using `gz` for now as we discussed ? 
Both libs are not something I'd like to rely on:
- On is 7 years old without changes since then
- The other has the following disclaimer: `The package is currently under development. There might be bugs and APIs are not considered stable`

cc @BaptisteFoy @chouquette

Reply:
Agreed on xi2, but ulikunitz's `xz` is used in quite big projects:
- [Official GCP Terraform modules](https://github.com/GoogleCloudPlatform/terraform-google-secured-data-warehouse)
- [Kubescape](https://github.com/kubescape/kubescape)
- Container runtimes such as [Rancher](https://github.com/rancher/rancher), [Portainer](https://github.com/portainer/portainer), and [Kata containers](https://github.com/kata-containers/kata-containers)
- [K9s](https://github.com/derailed/k9s)
- [Snyk](https://github.com/snyk/cli)

And many more. 

I'll at least replace xi2s' by ulikunitz's